### PR TITLE
/book-demo iteration

### PR DESCRIPTION
**Pull request by Yndira-FlowForge**
## Description

Added social proof and links to other contact channels

## Related Issue(s)

#1385

## Checklist

 - [x] I have read the [contribution guidelines](https://github.com/FlowFuse/flowforge/blob/main/CONTRIBUTING.md)
 - [x] I have considered the performance impact of these changes
 - [ ] Suitable unit/system level tests have been added and they pass
 - [ ] Documentation has been updated

### CODE REVIEW

File: src/book-demo.njk [hunk 1]
```diff
@@ -8,12 +8,64 @@ meta:
     description: Book a demo to understand how FlowFuse helps you professionalize your Node-RED deployment.
 ---
 <div class="text-left max-w-full ff-bg-light">
-  <div class="flex justify-center items-center md:block">
-    <div class="ff-bg-light w-full product px-4 pt-4 md:pt-12">
+  <div class="flex flex-col justify-center items-center md:block">
+    <div class="ff-bg-light w-full product px-4 pt-4 md:pt-12 -mb-24 sm:-mb-6">
```
Comment (ZJvandeWeg) [inline src/book-demo.njk:12]: Does tailwind have a way to collect classed together into one with a semantical naming scheme?

File: src/book-demo.njk [hunk 2]
```diff
@@ -8,12 +8,64 @@ meta:
     description: Book a demo to understand how FlowFuse helps you professionalize your Node-RED deployment.
 ---
```
Comment (ZJvandeWeg) [inline src/book-demo.njk:9]: Now the pages are so alike, shouldn't it be a one layout and two pages that render them?

Comment (ZJvandeWeg) [reply src/book-demo.njk:9]: Or alternatively, more partials?

File: src/book-demo.njk (hunk 1 above)
Comment (Yndira-FlowForge) [reply src/book-demo.njk]: I created new classes, it's cleaner now

File: src/book-demo.njk (hunk 2 above)
Comment (Yndira-FlowForge) [reply src/book-demo.njk:23]: Done, one layout and two pages

**Comment by Yndira-FlowForge** [comment]
@ZJvandeWeg 

> I think we can DRY the code out a bit

You were right, it's better now 😉

**Pull request merged**